qoqo_qasm/Cargo.toml

```diff
@@ -27,12 +27,12 @@ crate-type = ["cdylib", "rlib"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-qoqo = { version = "1.5", default-features = false }
-roqoqo = { version = "1.5", features = ["serialize"] }
+qoqo = { version = "1.5.1", default-features = false }
+roqoqo = { version = "1.5.1", features = ["serialize"] }
```

Comment:
Any reason for specifying the last digit? By writing just "1.5" it should automatically use the last available version